service/src/main/resources/logback.xml

```diff
@@ -25,6 +25,15 @@
         </encoder>
     </appender>
 
+    <if condition='isDefined("SENTRY_DSN")'>
```

Comment:
I think since we are just putting the DSN straight in the providers.ymp (which i think is fine) this condition may never be true? I would just remove it.

Reply:
This condition will be looking for an environment variable i think, not something from the spring profile

Reply:
Oh okay! I'm going to test this out again...I thought I didn't have this set as an env variable when I was running it locally